helm-repositories/kaptain/kaptain-repo.yaml

```diff
@@ -10,3 +10,5 @@ spec:
   url: "${helmMirrorURL:=http://kaptain-helm-mirror.kommander.svc}"
   interval: 10m
   timeout: 1m
+  upgrade:
```

Comment:
Should we set it for install as well so we replace the old CRD with the new one? Might be useful for the long time living clusters, that might still have old Kaptain CRD left after previous installations

Reply:
I think Install is the install event hook. I set `upgrade` to `CreateReplace` so it should create new and replace existing CRDs on a helm upgrade. This would leave the CRDs to be created be helm when we do a new install. Given that we have them in CRD folders and it's working today, I didn't want to change the way we create CRDs on the initial install.

Reply:
> Given that we have them in CRD folders and it's working today, I didn't want to change the way we create CRDs on the initial install.

It is working today on a _fresh_ install - this might not be the case for all the installations. What about migration? What if you try to install it on the K8s cluster and there are still CRDs left from the previous installation or in a case when the existing CRDs need to be preserved to save the workloads and we need to replace them during the install hook? I think it's safe to use `CreateReplace` policy for `install` as well just to be sure there are no stale CRDs left during the installation

Reply:
Makes sense. Now to figure out if there is a way to add this to HelmRelease in Kommander.